test/integration/scheduler_perf/scheduler_perf_test.go

```diff
@@ -732,6 +735,93 @@ func BenchmarkPerfScheduling(b *testing.B) {
 	}
 }
 
+var testSchedulingLabelFilter = flag.String("test-scheduling-label-filter", "integration-test", "comma-separated list of labels which a testcase must have (no prefix or +) or must not have (-)")
```

Comment:
Can we make the usage string a little different with the `perfSchedulingLabelFilter`, saying, this is integration test and the other is perf test?